include/irr/core/math/aabb2dSIMDIntBase.h

```diff
@@ -24,6 +23,9 @@ class aabbSIMDBase : public AlignedBase<_IRR_VECTOR_ALIGNMENT>
 		"assertion failed: cannot use this type");
 
 public:
+	aabb2dSIMDBase() = default;
```

Comment:
make them protected, this way `aabb2dSIMDBase` will not be possible to instantiate

Reply:
makes sense